app/src/main/java/com/example/recipeapp/Fragments/SearchFragment.java

```diff
@@ -136,7 +137,8 @@ private void searchListener() {
         RecipeApi service = RetrofitClientInstance.getRetrofitInstance().create(RecipeApi.class);
         Call<Envelope<List<Recipe>>> call = service.getRecipesWithFilters(BuildConfig.API_KEY, putWithEmptyCheck(binding.etTitle.getText()), cuisine, excludeCuisine,
                 putWithEmptyCheck(String.join(",", included)), putWithEmptyCheck(String.join(",", excluded)),
-                putWithEmptyCheck(binding.spinnerType.getSelectedItem()), putWithEmptyCheck(binding.edTime.getText().toString()), RecipeApi.RECIPE_INFORMATION_VALUE);
+                putWithEmptyCheck(binding.spinnerType.getSelectedItem()), putWithEmptyCheck(binding.edTime.getText().toString()),
+                Settings.getIntolerances(), Settings.getDiet(), RecipeApi.RECIPE_INFORMATION_VALUE);
 
```

Comment:
ViewModel

Reply:
I'm already doing it in the SaveOffline, so I could update it afterwards